integreat_cms/cms/models/pois/poi.py

```diff
@@ -87,6 +90,15 @@ class POI(AbstractContentModel):
             _("The opening hours remain and are only hidden."),
         ),
     )
+    appointment_url = models.URLField(
+        max_length=500,
+        blank=True,
+        null=True,
```

Comment:
```suggestion
```
It doesn't make a huge difference, but I think `null=True` for char fields rarely makes sense, since it's only `None` until the form has been submitted once, after that it's an empty string.
So it leads to a bit of an inconsistent database where some fields are `None` and some fields are `""` when the URL is not set.
Anyways, not super important, just something to keep in mind I guess.